packages/php/README.md

```diff
@@ -0,0 +1,292 @@
+                echo "Batch progress: " . round($progress * 100) . "%\n";
+            }
+        ]);
+
+        $localizedTexts = $lingoEngine->batchLocalizeText($textsToLocalize, $targetLocale);
+
+        echo "Batch Localized Texts (fr):\n";
+        print_r($localizedTexts);
+        /* Output (example):
+        Batch progress: 0%
+        Batch progress: 100%
+        Batch Localized Texts (fr):
+        Array
+        (
+            [0] => Bonjour
+            [1] => Au revoir
+            [2] => Merci
+        )
+        */
+
+    } catch (\Exception $e) {
+        echo "Error batch localizing texts: " . $e->getMessage() . "\n";
+    }
+    ```
+
+*   **Localize Chat:**
+
+    ```php
+    try {
+        $chatMessages = [
+            ['role' => 'system', 'content' => "You are a helpful assistant."],
+            ['role' => 'user', 'content' => "What is the capital of France?"],
+            ['role' => 'assistant', 'content' => "The capital of France is Paris."]
+        ];
+        $targetLocale = "ja"; // Target locale (e.g., Japanese)
+
+        $localizedChat = $lingoEngine->localizeChat($chatMessages, $targetLocale);
+
+        echo "Localized Chat (ja):\n";
+        print_r($localizedChat);
+        /* Output (example):
+        Localized Chat (ja):
+        Array
+        (
+            [0] => Array
+                (
+                    [role] => system
+                    [content] => あなたは役に立つアシスタントです。
+                )
+            [1] => Array
+                (
+                    [role] => user
+                    [content] => フランスの首都はどこですか？
+                )
+            [2] => Array
+                (
+                    [role] => assistant
+                    [content] => フランスの首都はパリです。
+                )
+        )
+        */
+
+    } catch (\Exception $e) {
+        echo "Error localizing chat: " . $e->getMessage() . "\n";
+    }
+    ```
+
+**4. Error Handling**
+
+All SDK methods can throw exceptions (e.g., `\GuzzleHttp\Exception\RequestException` for network issues, `\Exception` for API errors or configuration problems). Wrap your SDK calls in `try...catch` blocks to handle potential errors gracefully.
+
+## Configuration Options
+
+When creating the `LingoDotDevEngine` instance, you can pass an associative array with the following options:
+
+*   `apiKey` (string, required): Your Lingo.dev API key.
+*   `apiUrl` (string, optional): The base URL for the Lingo.dev API. Defaults to `https://api.lingo.dev`.
+*   `timeout` (int, optional): Request timeout in seconds. Defaults to 30.
+*   `retries` (int, optional): Number of retry attempts on failure. Defaults to 3.
+*   `onProgress` (callable, optional): A callback function that receives a float between 0.0 and 1.0 indicating the progress of batch operations. `function($progress) { ... }`.
+
+You can also update the configuration after instantiation using the `setConfig` method:
+
+```php
+$lingoEngine->setConfig([ 'timeout' => 60 ]);
```

Comment:
The `setConfig` method is not implemented. Also the `timeout` seems to be hardcoded, not configurable.